addons/binding/org.openhab.binding.evohome/src/main/java/org/openhab/binding/evohome/handler/EvohomeAccountBridgeHandler.java

```diff
@@ -117,10 +123,24 @@ public LocationsStatus getEvohomeStatus() {
         return apiClient.getInstallationStatus();
     }
 
+    public DailySchedules getZoneSchedule(String zoneType, String zoneId) {
```

Comment:
Ended up going down creating a seperate tryToCall with RunnableWithReturnAndTimeout to handle this case. On the plus side it can now be used in a few more cases.